bin/ch-image.py.in

```diff
@@ -251,6 +251,10 @@ def main():
    add_opts(sp, misc.list_, deps_check=True, stog_init=True)
    sp.add_argument("-l", "--long", action="store_true",
                    help="use long listing format")
+   sp.add_argument("-u", "--undeleteable", action="store_true",
+                   help="list images that can be restored with “undelete”")
+   sp.add_argument("--undeletable", action="store_true",
+                   help=argparse.SUPPRESS)
```

Comment:
With an additional argument, this can store in the same attribute as the documented one.